Update Dashboard page documentation (#999)
returntocorp/semgrep-docs: docs/semgrep-cloud-platform/dashboard.md

![Screenshot of dashboard view](/img/dashboard-view.png)

The Semgrep Cloud Platform Dashboard is an overview of your organization’s security posture from data aggregated within the Semgrep Cloud Platform. With these metrics you can:

* View recurring security issues, consequently taking action on them.
* Improve communication between developer teams and security teams.
* Detect vulnerabilities early, thereby preventing these from persisting through to the next stage of product delivery, such as QA.

You can access the Dashboard by [logging into the Semgrep Cloud Platform](https://semgrep.dev/login?return_path=/manage/projects).

## Assessing security readiness at a glance

![Screenshot of dashboard Code widget](/img/dashboard-code.png)

> The **Code** widget displays high-level security analytics across your entire organization. This includes:

Reviewer: Just FYI, the "Code" and "Supply Chain" titles are titles of the two sides of the page. The header widgets have no title of their own. You can choose to call them Code widget / Supply Chain widget if you want.

<dl>
	<dt>High severity</dt>
	<dd>Findings generated by a rule with the severity value set to <code>ERROR</code>. These include security backdoors, and highly vulnerable code. If you filter for another time period than <b>All time</b> the displayed number badge compares the number of high-severity findings within the given time period against the previous time period.</dd>
	<dt>Open findings</dt>
	<dd>The number of open findings over the <b>given time period</b>. The badge number indicates whether this number has gone up or down compared to the previous timeframe. If you filter for another time period than <b>All time</b> the displayed number badge compares the most recent number of open findings against the previous timeframe.</dd>
	<dt>Comment fix rate</dt>
	<dd>The percentage of findings that were fixed when findings surfaced to developers through PR or MR comments in previous scans. If you filter for another time period than <b>All time</b> the displayed number badge compares PR and MR fix rate in the given time period against the previous time period.</dd>
</dl>

## Filtering findings by time

> The Dashboard displays data from scans for the **last 1 month** by default. This time range can be set to a broader or narrower value. By broadening the time range, security teams are able to see total numbers and statistics across an entire time period. Narrow time ranges can give insights into the most recent vulnerabilities creeping into the project.

Reviewer 2: Not sure if it is still one month. I checked and it's set to "All time".

Author: @s-santillan I was not sure if I see All time because I selected it once...

```suggestion
The Dashboard displays data from scans for the **All time** by default. This time range can be set to a narrower value. By broadening the time range, security teams are able to see total numbers and statistics across an entire time period. Narrow time ranges can give insights into the most recent vulnerabilities creeping into the project.
```

The **Rules summary** widget provides a summary report for rule metrics, such as what rules are ignored or fired the most.

These data points can serve as a starting point for the following security audits:

- Investigating the relevance or quality of a rule. For example: Is this rule useful, or does it detect too many false positives?
- Are there underlying issues in the codebase that result in recurring patterns of insecure code?
- Are there rules that developers don’t resolve? Semgrep helps identify such rules, which helps to form insights into possible causes.

## Using Dashboard with Semgrep Supply Chain

Semgrep Dashboard can display vulnerable dependency findings of Semgrep Supply Chain.

![Screenshot of dashboard with Semgrep Supply Chain](/img/dashboard-with-supply-chain.png)

Semgrep Supply Chain dashboard consists of three widgets:
<dl>
	<dt>Supply Chain</dt>
	<dd>Contains three items: <b>Reachable vulns</b>, <b>Unreachable vulns</b>, and <b>Undetermined vulns</b>.</dd>
	<dt>Most vulnerabilities</dt>
	<dd>The number of dependency vulnerabilities over the given time period next to the calendar icon.</dd>
	<dt>New advisories</dt>

> 	<dd>Announcements of new vulnerabilities associated with dependencies in your projects.</dd>

Reviewer: ```suggestion
	<dd>Announcements of new vulnerabilities.</dd>
```

1. Click the <i class="fa-solid fa-sliders"></i> **All projects** button.
2. Select the project(s) from the drop-down box. The Dashboard, including all widgets, reloads to reflect data from the selected project(s).

> ## Summarizing a projects security posture

Reviewer 2: What's the reason for this edit? I don't think it makes any sense?

Author: ```suggestion
## Summarizing a project security posture
```

Reviewer 2: Sure go ahead and commit!

The **Most findings** widget displays open findings, high severities, and fix rates per-project. Through this view, you can see a specific number of findings in given projects. The columns are arranged in descending order, from the project with the greatest amount of findings to the least.

To view the project’s findings, click on the project’s name. This takes you to the [Findings page](/semgrep-code/findings), where you can filter, sort, and triage findings.

## Assessing rule performance

![Screenshot of dashboard rules widget](/img/dashboard-rules.png)

The **Rules summary** widget provides a summary report for rule metrics, such as what rules are ignored or fired the most.

These data points can serve as a starting point for the following security audits:

- Investigating the relevance or quality of a rule. For example: Is this rule useful, or does it detect too many false positives?
- Are there underlying issues in the codebase that result in recurring patterns of insecure code?
- Are there rules that developers don’t resolve? Semgrep helps identify such rules, which helps to form insights into possible causes.

## Using Dashboard with Semgrep Supply Chain

> :::info Prerequisite

Reviewer 2: ```suggestion
```
I think it's safe to delete this because SSC is enabled by default now.

> To display Semgrep Supply Chain findings in Semgrep Dashboard enable Semgrep Supply Chain. See the [Getting started with Semgrep Supply Chain](/semgrep-supply-chain/getting-started/) article for more information.

Reviewer 2: ```suggestion
```

> :::

Reviewer 2: ```suggestion
```

![Screenshot of developer feedback end flow](/img/dashboard-devfeedback2.png)

-->

> ## See also

Reviewer 2: Any reason for the removal?

Reviewer 2: Ah, because it was moved up. Resolving.
